AndorsTrail/res/raw/conversationlist_glasforn_fixes.json

```diff
@@ -0,0 +1,42 @@
+[
+    {
+        "replies":[
+            {
+                "text":"N",
+                "nextPhraseID":"glasforn_rumblings60_5_3"
+            }
+        ],
+        "id":"glasforn_rumblings60_5_2",
+        "message":"I couldn\u2019t. When I tried, and failed, he laughed. He told me it was bound to that thing in the basement, which needed me to be its servant. "
+    },
+    {
+        "replies":[
+            {
+                "text":"N",
+                "nextPhraseID":"glasforn_rumblings60_6"
+            }
+        ],
+        "id":"glasforn_rumblings60_5_4",
+        "message":"I didn\u2019t dare. The evening after Andor left his companion returned briefly. He warned me you might come here, and that if you found out what I had done you would kill me. He told me that my only hope was to make you the next victim."
+    },
+    {
+        "replies":[
+            {
+                "text":"I guess you are lucky I visited, although just asking me to kill it would have been easier. And nicer.",
+                "nextPhraseID":"glasforn_rumblings60_5_4"
+            }
+        ],
+        "id":"glasforn_rumblings60_5_3",
+        "message":"Then he laughed some more, and told me that all I had to do was kill that thing, and I would able to remove the necklace. I\u2019m no fighter though. I was too scared to even go near it."
```

Comment:
The unicode escapes and the negated requirements are very strange. I have no idea how they got there. I'll search for them, and make sure they are all gone.